modules/products/jira/variables.tf

```diff
@@ -67,27 +67,59 @@ variable "jira_configuration" {
   }
 }
 
+variable "nfs_requests_cpu" {
+  description = "The minimum CPU compute to request for the NFS instance"
+  type        = string
+  default     = "1"
```

Comment:
Do we need to include defaults on this level as well if they are the same as the defaults in the `AWS/nfs` module?